.travis.yml

```diff
@@ -4,7 +4,19 @@ os:
 language: shell
 services: docker
 env:
+- HHVM_VERSION=3.28.3
```

Comment:
Should we start from `3.25.3` like in [fb_intercept_polyfill](https://github.com/lexidor/fb_intercept_polyfill/blob/ef93b77e4753feefc77206ec9a954fde4c318929/.travis.yml#L5)?

Reply:
Yes, but that requires making the same changes to the Travis dot yaml as fb_intercept_polyfill.
I did have this done by then.
I'll see if it can be copy pasted verbatim.